### PR TITLE
created authorization for taxi service

### PR DESCRIPTION
**Pull request by Timur5050**

### CODE REVIEW

**Comment by Timur5050** [comment]
Снимок экрана 2024-07-04 в 12 27 35
Снимок экрана 2024-07-04 в 12 27 43
Снимок экрана 2024-07-04 в 12 27 47
Снимок экрана 2024-07-04 в 12 27 50

File: templates/registration/logged_out.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h1>Logout</h1>
+<a href="{% url 'login' %}">to login click here</a>
+
+{% endblock %}
```
Comment (Dimosphen1) [inline templates/registration/logged_out.html:7]: Add blank lines at the end of the files where indicated

File: db.sqlite3
Comment (Dimosphen1) [inline db.sqlite3:1]: Do not commit DB files

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

**Comment by Timur5050** [comment]
but i have this, check index function and index.html

**Review by EdAlekseiev — APPROVED** [review]

**Pull request closed without merging**